src/SevenSegmentTM1637.h

```diff
@@ -20,7 +20,13 @@
  #include <WProgram.h>
 #endif
 
-#include <avr/pgmspace.h>   // Used for PROGMEM
+//#include <avr/pgmspace.h>   // Used for PROGMEM
```

Comment:
What's the point of this comment if you just have that code repeated 3 lines down?